Comp2023-Imported/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -186,12 +186,16 @@ public void updateSwerveOdometry( )
 
       //Adding a position specified by the limelight to the estimator at the time that the pose was generated 
       if (botLLPose != null && DriverStation.isTeleopEnabled( ))
+      {
         m_poseEstimator.addVisionMeasurement(botLLPose, Timer.getFPGATimestamp( ) - (latency / 1000));
+      }
 
       Pose2d rawPose = m_vision.getLimelightRawPose( );
+      Pose2d Origin = new Pose2d(new Translation2d(0.0, 0.0), new Rotation2d(0));
```

Comment:
This is a minor "nit", but origin is a local variable and should be "origin" (drinking camel case).

Reply:
I guess this is in Comp2023-imported anyway and not our Comp2024 codebase.